src/main/java/seedu/address/logic/parser/RemoveCommandParser.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ATTENDANCE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_GRADEPROGRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_HOMEWORK;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SESSION;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.RemoveCommand;
+import seedu.address.logic.commands.RemoveCommand.RemovePersonDescriptor;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new RemoveCommand object
+ */
+public class RemoveCommandParser implements Parser<RemoveCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the RemoveCommand
+     * and returns the RemoveCommand object for execution
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public RemoveCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_HOMEWORK,
+                        PREFIX_GRADEPROGRESS, PREFIX_ATTENDANCE, PREFIX_SESSION);
+
+        Index index;
+
+        try {
+            index = ParserUtil.parseIndex(argMultimap.getPreamble());
+        } catch (ParseException pe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, RemoveCommand.MESSAGE_USAGE), pe);
+        }
```

Comment:
there isn't supposed to be a person index for the remove command

Reply:
the edit command has a person index though

Reply:
yea that's why it's my job to update it LOL

Reply:
oh LOL
pls update mine too thx

Reply:
that was  a jk btw

Reply:
u will be cursed with a CI that never passes